sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerAccessor.java

```diff
@@ -17,19 +17,36 @@
  */
 package org.apache.beam.sdk.io.gcp.spanner;
 
+import com.google.api.gax.rpc.FixedHeaderProvider;
+import com.google.cloud.ServiceFactory;
 import com.google.cloud.spanner.BatchClient;
 import com.google.cloud.spanner.DatabaseAdminClient;
 import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
 import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.cloud.spanner.spi.v1.SpannerInterceptorProvider;
+import io.grpc.CallOptions;
+import io.grpc.Channel;
+import io.grpc.ClientCall;
+import io.grpc.ClientInterceptor;
+import io.grpc.MethodDescriptor;
+import java.util.concurrent.TimeUnit;
+import org.apache.beam.sdk.options.ValueProvider;
+import org.apache.beam.sdk.util.ReleaseInfo;
+import org.joda.time.Duration;
 
 /** Manages lifecycle of {@link DatabaseClient} and {@link Spanner} instances. */
 public class SpannerAccessor implements AutoCloseable {
```

Comment:
SpannerAccessor is public. Therefore any public methods' return values may become the source of the API surface leak.